### PR TITLE
feat: Created Image and card component

### PR DESCRIPTION
**Pull request by lakshaychopra**

### CODE REVIEW

**Review by someshtiwarii — APPROVED** [review]
you can remove the commented code, rest looks good!

File: components/card/card.css [hunk 1]
```diff
@@ -0,0 +1,127 @@
...
+    margin: 0.5rem;
+    height: 100%;
+}
+
+/* .card-btns {
+    margin: 0.5rem;
+    display: flex;
+    align-items: center;
+    gap: 1rem;
+} */
```
Comment (manojasarsa) [inline components/card/card.css:46]: You can remove this commented part, if no longer in use.

**Review by manojasarsa — APPROVED** [review]
Rest looks good!

**Comment by preethasri** [comment]
looks good

**Pull request merged**